src/UI/Internal/Tables/View.elm

```diff
@@ -47,6 +52,26 @@ simpleText renderConfig text =
             ]
 
 
+linkedText : RenderConfig -> Link -> Text -> Element msg
+linkedText renderConfig link text =
+    text
+        |> Text.withOverflow ellipsizeWithTooltip
+        |> Text.withColor (Palette.color tonePrimary brightnessMiddle)
+        |> Text.renderElement renderConfig
```

Comment:
what if we need links with the overflow behavior outside the table? We'd have to copy it over, so maybe we should create a standalone link component outside of the button. Just something to keep in mind

Reply:
I find it interesting, could you propose the API for that? Also, CC @jouderianjr

Reply:
yeah, taking in mind the overflow, it makes sense to have it also as a component

Reply:
We can do it after the BF thing, can you create a card for that @PedroHLC ?